### PR TITLE
Fix "dnu wrap" help information

### PR DESCRIPTION
**Pull request by ChengTian**
`dnu wrap` can wrap dlls. Fix the help information to make users aware of this.

@davidfowl @muratg

### CODE REVIEW

File: src/Microsoft.Framework.PackageManager/Program.cs [hunk 1]
```diff
@@ -481,9 +481,9 @@ public int Main(string[] args)
 
             app.Command("wrap", c =>
             {
-                c.Description = "Wrap a csproj into a project.json, which can be referenced by project.json files";
+                c.Description = "Wrap a csproj/dll into a project.json, which can be referenced by project.json files";
```
Comment (muratg) [inline src/Microsoft.Framework.PackageManager/Program.cs]: Does "assembly" make more sense than dll?

Comment (ChengTian) [inline src/Microsoft.Framework.PackageManager/Program.cs]: Yeah, "assembly" sounds more formal. Fixed

**Comment by troydai** [comment]
:shipit:

**Comment by muratg** [comment]
:shipit:

**Pull request merged**